Add governance model; update roadmap (#9388)
mne-tools/mne-python: doc/overview/governance.rst

documentation, designs, or other work to the Project. Anyone can be a
Contributor. Contributors can be affiliated with any legal entity or
none. Contributors participate in the project by submitting, reviewing,
and discussing GitHub Pull Requests and Issues and participating in open
and public Project discussions on GitHub, Discourse, and other
channels. The foundation of Project participation is openness and
transparency.

The Project Community consists of all Contributors and Users of the
Project. Contributors work on behalf of and are responsible to the
larger Project Community and we strive to keep the barrier between
Contributors and Users as low as possible.

The Project is not a legal entity, nor does it currently have any formal
relationships with legal entities.

Governance model
================

This section describes the governance and leadership model of The
Project.

The foundations of Project governance are:

-  openness and transparency
-  active contribution
-  institutional neutrality

Traditionally, Project leadership was provided by a subset of Contributors,
called Core Developers, whose active and consistent contributions have been

> recognized by their receiving “commit rights” to the Project GitHub

Reviewer: `by their receiving “commit rights” to the Project GitHub` sounds a bit awkward to me - as a non-native English speaker I stumbled across it while reading. Maybe there is a better way to phrase it?

Author: ```suggestion
Traditionally, Project leadership was provided by a subset of Contributors,
informally called Core Developers, whose active and consistent contributions
were rewarded by granting them “commit rights” to the Project GitHub
```
is this better @britta-wstnr ?

> Traditionally, Project leadership was provided by a subset of Contributors,
> called Core Developers, whose active and consistent contributions have been
> recognized by their receiving “commit rights” to the Project GitHub

Author: ```suggestion
Traditionally, Project leadership was provided by a subset of Contributors,
informally called Core Developers, whose active and consistent contributions
were rewarded by granting them “commit rights” to the Project GitHub
```
better?
